bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -121,7 +121,7 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
                             new StateOption("2", "average"), new StateOption("3", "good"),
                             new StateOption("4", "excellent")))
                     .build())
-            .withTags(List.of("Measurement", "Level")).build();
+            .withTags(List.of("Measurement", "SignalStrength")).build();
```

Comment:
WDYT of this:

```suggestion
            .withTags(List.of(DefaultSemanticTags.Point.MEASUREMENT.getName(), DefaultSemanticTags.Property.SIGNAL_STRENGTH.getName())).build();
```

Reply:
We could import DefaultSemanticTags.* to shorten them to just `Point.MEASUREMENT.getName()`

Reply:
Yes that is an excellent idea. It would encourage others to keep the same semantics in future.